docs/src/app/components/pages/components/List/SelectableInstruct.jsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import Paper from 'material-ui/lib/paper';
+import Typography from 'material-ui/lib/styles/typography';
+import CodeBlock from '../../../CodeExample/CodeBlock';
+
+const styles = {
+  headline: {
+    fontSize: '24px',
+    lineHeight: '32px',
+    paddingTop: '16px',
+    marginBottom: '12px',
+    letterSpacing: '0',
+    fontWeight: Typography.fontWeightNormal,
+    color: Typography.textDarkBlack,
+  },
+  subheadline: {
+    fontSize: 18,
+    lineHeight: '27px',
+    paddingTop: 12,
+    marginBottom: 9,
+    letterSpacing: '0',
+    fontWeight: Typography.fontWeightNormal,
+    color: Typography.textDarkBlack,
+  },
+  codeblock: {
+    padding: 24,
+    marginBottom: 32,
+  },
+};
+
+const SelectableInstruct = () => (
```

Comment:
Can we use the `MarkdownElement`. I think that it would be simpler to read and to maintain :smiley: